README.md

```diff
@@ -120,7 +120,15 @@ Install the Go compiler from [golang.org](https://golang.org/). And make sure yo
  git clone https://github.com/noisetorch/NoiseTorch # Clone the repository
  cd NoiseTorch # cd into the cloned repository
  make # build it
- ```
+```
+
+To install it:
+
+```shell
+cp ./bin/noisetorch ~/.local/bin
+cp ./assets/noisetorch.desktop ~/.local/share/applications
+cp ./assets/icon/noisetorch.png ~/.local/share/icons/hicolor/256x256/apps
+```
```

Comment:
I'd like you to move it just above the `Uninstall` section. I think that would be the best place to put this section. What do you think?

Reply:
Doesn't the "Download & Install" section refers to an installation from binary release?

Reply:
Oh you're right. Sorry.

Reply:
I can't mark this thread as resolved sadly, but I approved your changes.